src/extensibility/node/package-validator.js

```diff
@@ -232,6 +232,29 @@ function validatePackageJSON(path, packageJSON, options, callback) {
                 }
             }
             
+            // normalize translated author and contributors
+            if (metadata["package-i18n"]) {
+                Object.keys(metadata["package-i18n"]).forEach(function (lang) {
+                    var packageLang = metadata["package-i18n"][lang];
+
+                    if (packageLang.author) {
```

Comment:
This block (down to line 254) is duplicated with the code above, so please factor it into a reusable function.